CMakeLists.txt

```diff
@@ -623,4 +618,4 @@ if (CMAKE_GENERATOR STREQUAL "Ninja Multi-Config" AND PROJECT_IS_TOP_LEVEL)
    set (CMAKE_CROSS_CONFIGS "all")
 endif ()
 
-feature_summary(WHAT ENABLED_FEATURES DISABLED_FEATURES)
+feature_summary(WHAT ALL)
```

Comment:
```suggestion
feature_summary(WHAT ENABLED_FEATURES DISABLED_FEATURES)
```

Revert?

Testing configuring with:
```
cmake \
    -G "Ninja" \
    -DENABLE_SSL=DARWIN \
    -S$(pwd) -B$(pwd)/cmake-build
```

Before these changes shows:

```
-- The following features have been enabled:

 * TLS, for secure network communication (SecureTransport)
 * Cryptography, cryptographic primitives (SecureTransport)
 * AWS Authentication, authenticate with MongoDB servers using credentials from AWS instance metadata
 * SASL Authentication, authenticate with MongoDB servers using SASL: “Simple Authentication and Security Layer” (Cyrus)

-- Configuring done (0.8s)
```

After these changes:
```
-- The following features have been enabled:

 * TLS, for secure network communication (SecureTransport)
 * Cryptography, cryptographic primitives (SecureTransport)
 * AWS Authentication, authenticate with MongoDB servers using credentials from AWS instance metadata
 * SASL Authentication, authenticate with MongoDB servers using SASL: “Simple Authentication and Security Layer” (Cyrus)

-- The following OPTIONAL packages have been found:

 * Python
 * Python3
 * PkgConfig
 * mongocrypt

-- The following REQUIRED packages have been found:

 * SASL2 (required version >= 2.0), Cyrus implementation of SASL
   Provides libmongoc SASL authentication support
 * Threads
 * Sphinx

-- The following REQUIRED packages have not been found:

 * Utf8Proc

-- Configuring done (0.7s)
```

The listing of `Utf8Proc` as "REQUIRED" and "not found" seems unexpected. That may be surfacing a separate issue. Until that is addressed, I suggest excluding from the feature summary.

Reply:
I forgot about the misleading `Utf8Proc` report, which is why I didn't use `ALL` in the prior PR. I can see the cause and opened a ticket for this one: CDRIVER-4732